source/Calamari.Azure.CloudServices/Deployment/Conventions/ChooseCloudServiceConfigurationFileConvention.cs

```diff
@@ -58,7 +58,7 @@ string GetFirstExistingFile(RunningDeployment deployment, params string[] fileNa
         static string BuildEnvironmentSpecificFallbackFileName(RunningDeployment deployment)
         {
             return string.Format(EnvironmentFallbackFileName,
-                deployment.Variables.Get(SpecialVariables.Environment.Name));
+                deployment.Variables.Get(Common.Variables.EnvironmentVariables.Name));
```

Comment:
Looked into why this has to be namespaced, and `Calamari.Shared\EnvironmentVariables` should be called something else I think

Reply:
Would `KnownEnvironmentVariables` clean this up?

Reply:
perhaps `DeploymentEnvironmentVariables`?